model/infinispan/src/main/java/org/keycloak/connections/infinispan/DefaultInfinispanConnectionProviderFactory.java

```diff
@@ -310,7 +310,9 @@ protected void initEmbedded() {
             configureRemoteCacheStore(replicationConfigBuilder, async, InfinispanConnectionProvider.WORK_CACHE_NAME, false);
         }
 
-        Configuration replicationEvictionCacheConfiguration = replicationConfigBuilder.build();
+        Configuration replicationEvictionCacheConfiguration = replicationConfigBuilder
+          .expiration().enableReaper().wakeUpInterval(15, TimeUnit.SECONDS)
```

Comment:
Just curious about this change - seems the reaper is active by default and runs every 60 seconds. What would be the benefits of changing this to every 15 seconds?

Reply:
This is a change only in the embedded, so the testsuite-only change, and only limited to `work` cache. The benefit is to have the tests run faster, rather than waiting 60 seconds for expiration thread to wake up.

Reply:
Got it, thanks